sys/include/net/ng_pktbuf.h

```diff
@@ -57,33 +57,37 @@ extern "C" {
 #endif  /* NG_PKTBUF_SIZE */
 
 /**
+ * @brief   Initializes packet buffer module.
```

Comment:
From the ones I wrote that come to mind on the top of my mind: basically every `*_netif` module, `ng_sixlowpan_ctx`, ... many more… basically anything that has an internally stored data structure (including Martin's FIP)... I guess ^^.